field.field.config_pages.stanford_basic_site_settings.su_site_type.yml

```diff
@@ -0,0 +1,21 @@
+uuid: 18918dc6-5dca-49ee-be52-a2d0c57ac172
```

Comment:
This file should be in the `config/sync` directory

Reply:
Fixed!